### ISSUE TITLE
Generate a black image when previewing

### ISSUE THREAD

**Issue by jyuan15** [Mikubill/sd-webui-controlnet#1004]
I reinstalled the latest version of controlnet, when previewing with openpose, a black image was generated, but cmd did not output any errors.
The graphics card is GTX1660s, and the SD boot parameters are --xformers --medvram --lowvram --precision full --no-half.

I have been unable to use controlnet, and I have reinstalled stable-diffusion-webui several times, but the problem is still not resolved, and I don't know what the reason is. hope someone can tell me, thanks!

![2023-04-24_080411](https://user-images.githubusercontent.com/12642186/233876902-8dcc5aac-02a8-4fa7-80d6-603be5c6af06.jpg)

**Comment by soyatlas** [comment]
me too :(

**Comment by jyuan15** [comment]
After checking, I found that it was caused by browser extensions.
I disabled all browser extensions, controlnet resumed working, then enabled extensions one by one until they were all enabled, controlnet still worked fine, so I don't know which extension caused it.
But, thank goodness, controlnet is finally working properly.
![2023-04-28_074833](https://user-images.githubusercontent.com/12642186/235014455-23271cfd-9673-4a24-8c0e-a9acca905a79.jpg)
